src/SprykerEco/Zed/PunchoutCatalogs/Presentation/EntryPoints/view.twig

```diff
@@ -0,0 +1,35 @@
+{% extends '@Gui/Layout/layout.twig' %}
```

Comment:
Please get approve from our FE-developer for this twig & all other FE-related things.